### PR TITLE
return and print infos

### PR DESCRIPTION
**Pull request by ChuckNorrison**
return generated informations to the function and not just print out

related to  #102

### CODE REVIEW

File: meshtastic/__init__.py [hunk 1]
```diff
@@ -227,8 +229,11 @@ def getTimeAgo(ts):
         for i, row in enumerate(rows):
             row['N'] = i+1
 
-        print(tabulate(rows, headers='keys', missingval='N/A',
-                       tablefmt='fancy_grid'), file=file)
```
Comment (ChuckNorrison) [inline meshtastic/__init__.py:231]: maybe someone should check for dependencies to the file=file property, i removed it on this

**Review by ChuckNorrison — COMMENTED** [review]
cant find any dependencies on this

File: meshtastic/__init__.py (hunk 1 above)
Comment (geeksville) [reply meshtastic/__init__.py:231]: Thanks for the PR.  I don't quite follow this part though, is there a reason you removed the option of emitting the showNodes() output to a destination other than stdout)?

Comment (ChuckNorrison) [reply meshtastic/__init__.py:231]: As i changed the prints to assignments, i felt this "file" output was not necessary on this methods, so i tested without them and it works.

Comment (geeksville) [reply meshtastic/__init__.py:231]: ok - sounds good!

**Pull request merged**